src/index.html

```diff
@@ -11,12 +11,112 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/src/index.html"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="logo__img"
+        />
+      </a>
+
+      <nav class="nav">
+        <li class="list">
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </ul>
+
+          <ul class="list__item">
+            <a
+              href="/src/index.html"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </ul>
+        </li>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. 'list' should be replaced with something more meaningful, and 'list__item' should be 'nav__item' to reflect its role within the navigation.